README.md

```diff
@@ -1,3 +1,43 @@
 # The Process
 
-Under construction.
+This repository contains the source code for docker images that are used in [citus testing](https://github.com/citusdata/citus/blob/master/.circleci/config.yml).  
+
+The [extbuilder](https://github.com/citusdata/the-process/tree/master/circleci/images/extbuilder) image is the core image that other jobs depend on in our tests. The [extbuilder](https://github.com/citusdata/the-process/tree/master/circleci/images/extbuilder):
+
+- Installs postgres 10 and 11 `pg_config`. We only need `pg_config` to generate citus artifacts.
+- Installs and tars the checked out citus version for the postgres versions. This is done so that the other jobs can install citus easily without the need to do `make install`.
```

Comment:
```suggestion
- Installs and tars the checked out citus version for the postgres versions. This is done so that the other jobs can install citus easily by untarring without the need to do `make install`.
```